private_sharing/forms.py

```diff
@@ -142,9 +142,10 @@ def clean_project_member_ids(self):
                 'Project member IDs are always 8 digits long.')
 
         # look up each ID in the database
-        project_members = DataRequestProjectMember.objects.filter(
-            project_member_id__in=project_member_ids,
-            message_permission=True)
+        project_members = (DataRequestProjectMember.objects
```

Comment:
Does `.filter_active()` remove all the ones that have revoked access?

Reply:
yup see https://github.com/OpenHumans/open-humans/blob/4998871fc51d6a4466030bf031a3f5f8a53b807a/private_sharing/models.py#L309